### ISSUE TITLE
More actions on budgets

### ISSUE THREAD

**Issue by RocketC31** [RocketC31/budget#35]
Budgets fonctionnality can just create a budget, but no delete or edit is available.
Vue is very simple. 

Maybe it will be nice to add delete/trash/edit for this.
And why not add emails for bilan

**Comment by sebastiaanspeck** [comment]
Maybe this one is nice to start working on next?

**Comment by RocketC31** [comment]
Yes we can, maybe like transaction we can clarify what we imagine for it ? 

- First of all, i think we can rework on the vue, because it's so.... simple. Maybe we can use same the color tag before text label. Rework a little on the progressbar... maybe the progress bar can have same color as tag color, and add possibility to edit and remove
- After that, the feature is poor on functionality. We can imagine an email sended if we overflow the budget for a specific tag. If for exemple i have set '30e per week for tag fun', if i add a new transaction in this category and if overflow my budget limit this will send me and email to say to me i have overflow... Or maybe make it directly on the app when i start to add transaction ? 
- Like spending email every friday, why not make a resume email for the budget respect ?

**Comment by sebastiaanspeck** [comment]
Sounds like a good idea! Will provide more information this evening

**Comment by sebastiaanspeck** [comment]
- Your suggestion of using the tag color is a good one, but I also found out that some colors look a bit off when using dark mode (Vrije tijd and Declaraties e.a.): <img width="764" alt="image" src="https://user-images.githubusercontent.com/12570668/203404202-bafacb46-96a9-4b77-8cdf-9d58d2772661.png">

- It definitely needs the option to have a trash like tags/transactions/recurrings.
- I think your first idea of sending emails is the best idea. You may want to overflow your budget, but want to get reminded.
- A nice summary of your budgets on a weekly basis seems fine to me. Maybe choose a different day, like Wednesday at 18:00, otherwise the weekly report/budget overview mix up in your mailbox.

**Comment by sebastiaanspeck** [comment]
We could also make the tags like this:
<img width="745" alt="image" src="https://user-images.githubusercontent.com/12570668/203408446-4ac1dc6a-9c50-46a3-97d1-85753ea6779c.png">

**Comment by RocketC31** [comment]
> We could also make the tags like this: <img alt="image" width="745" src="https://user-images.githubusercontent.com/12570668/203408446-4ac1dc6a-9c50-46a3-97d1-85753ea6779c.png">
> 
> To me, it still looks like a tag, but the odd invisibility with some colors is less in my opinion.

Nice ! I like it !
